modules/end-user-guide/pages/troubleshooting-webview-loading-error.adoc

```diff
@@ -0,0 +1,42 @@
+:_content-type: PROCEDURE
+:description: Troubleshooting Webview loading error
+:keywords: user-guide, troubleshooting-webview-loading-error
+:navtitle: Troubleshooting Webview loading error
+:page-aliases: .:troubleshooting-webview-loading-error.adoc
+
+[id="troubleshooting-webview-loading-error"]
+
+= Troubleshooting Webview loading error
+
+pass:[<!-- vale RedHat.Spelling = NO -->]
+
+If you use Microsoft Visual Studio Code - Open Source in an incognito or private browser window, you can encounter an issue with Webviews. The Webview is not loaded and an error message appears: *Error loading webview: Error: Could not register service workers*.
+
+The issue affects the following browsers:
+
+* Google Chrome in Incognito mode
+* Mozilla Firefox in Private Browsing mode
+
+[WARNING]
+====
+Mozilla Firefox Private Browsing mode link:https://bugzilla.mozilla.org/show_bug.cgi?id=1320796[does not support ServiceWorkers] necessary for Microsoft Visual Studio Code - Open Source to load Webviews. There is no workaround available.
+====
+
+.Fixing Webview loading error in Google Chrome Incognito mode
+
+Change the cookie settings in Google Chrome to fix the issue.
+
+.Procedure
+
+. Open *Settings*.
+. Select *Privacy and security*.
+. Click *Cookies and other site data*.
+. Select *Allow all cookies*.
```

Comment:
I assume the reader still has the option of not using a private window, so it'd be worth letting the reader know that not using a private window solves this problem too.

Another way of presenting all of this information is to turn it into a table like this:

```suggestion
.Dealing with the webview error in a private browsing window

[cols="25h,~"]
|===
| Browser | Workarounds

| Google Chrome
|
* Do not use an Incognito window.

* Go to *Settings* -> *Privacy and security* -> *Cookies and other site data* > *Allow all cookies*.

| Mozilla Firefox
|
* Do not use a private window.

* See link:https://bugzilla.mozilla.org/show_bug.cgi?id=1320796[Bug 1320796: Support ServiceWorkers in Private Browsing Mode].

|===

```

Reply:
> I assume the reader still has the option of not using a private window, so it'd be worth letting the reader know that not using a private window solves this problem too. Maybe something to confirm with the SME.

That's correct. As it's mentioned at the beginning of this doc, the issue is relevant for private/incognito modes only.

Reply:
I like the idea of presenting workarounds as a table :+1:

Reply:
I see your point but I think not using a private window in Firefox isn't really a workaround. That's why I used a warning instead. 
I implemented your suggestion to get rid of the list but I think I should keep it, actually. Maybe rephrase it a little bit to emphasize we don't know if other browsers have the same issue. Because a table with workarounds implies the same as a list, doesn't it? That we have tested the browsers and we have solutions for the error for each browser.

Reply:
Update: the incognito mode does more than just block cookies so even if you enable cookies, other things such as site data still won't be saved.